k8s/kubevirt-dynamic/virtualmachineinstancemigration.go

```diff
@@ -174,6 +198,16 @@ func (c *Client) unstructuredGetVMIMigration(
 	if err != nil {
 		return nil, fmt.Errorf("failed to get 'vmiName' from the vmi migration spec: %w", err)
 	}
+	// annotations
```

Comment:
This is to return additional information for labels and annotations as part of the `unstructuredGetVMIMigration` function?

Reply:
yes